cmd/nerdctl/main.go

```diff
@@ -134,6 +134,9 @@ func initRootCmdFlags(rootCmd *cobra.Command, tomlPath string) error {
 			return err
 		}
 	}
+
+    // Overriding config path for subcommands
+	rootCmd.PersistentFlags().String("config", tomlPath, "location of config file")
```

Comment:
Where is this referred?

Reply:
looks like this PR need this https://github.com/containerd/nerdctl/pull/707#discussion_r789811259

Reply:
Not sure. But this PR seems wrong anyway, because tomlPath has to be determined before parsing the toml.

Reply:
Yes I know. The idea is to emerge the tomlPath to the application via cobra.